Bricks/net/http/impl/server.h

```diff
@@ -58,9 +58,9 @@ const char kCRLF[] = "\r\n";
 const size_t kCRLFLength = strings::CompileTimeStringLength(kCRLF);
 const char kHeaderKeyValueSeparator[] = ": ";
```

Comment:
Is the space mandatory? Looks wrong. RFC:
`header-field   = field-name ":" OWS field-value OWS`
Optional WhiteSpaces must be removed during parsing.